bitnami/kafka/README.md

```diff
@@ -479,6 +480,7 @@ Note: This option requires creating RBAC rules on clusters where RBAC policies a
 ```console
 externalAccess.enabled=true
 externalAccess.service.type=NodePort
+externalAccess.autoDiscovery.enabled=true
```

Comment:
Auto-discovery should be only enabled for "Option A". This option implies that the user is manually providing the list of node ports.